### PR TITLE
Positive on the Cookie Store API (subset)

### PR DESCRIPTION
**Pull request by jesup**

### CODE REVIEW

**Review by bakulf — APPROVED** [review]

**Review by tantek — CHANGES_REQUESTED** [review]
Update makes sense given comments by @martinthomson and @ehsan on #94. 

Given that this "positive" is only on a subset of the proposal, we should be precise in exactly what that subset is, and in asking for it, by filing an issue on https://github.com/WICG/cookie-store/issues/ asking for the precise subsetting we want (and believe others will support), and then citing that issue in our positive comment on #94 and/or the 'mozPositionDetail' field in this entry.

**Review by bakulf — APPROVED** [review]

**Comment by bakulf** [comment]
See https://github.com/mozilla/standards-positions/pull/1106

File: activities.json [hunk 1]
```diff
@@ -471,8 +471,8 @@
     "description": "An asynchronous Javascript cookies API for documents and workers",
     "id": "cookie-store",
     "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1475599",
-    "mozPosition": "defer",
-    "mozPositionDetail": "This API provides better access to cookies. However, the improvements also expand access to cookie metadata and the interactions with privacy features like the Storage Access API are unclear. Work on improving cookie interoperability, which is ongoing, might be needed before an assessment can be made.",
+    "mozPosition": "positive",
+    "mozPositionDetail": "This API provides better access to cookies. Due to privacy concerns as outlined in issue 94, we are positive for a subset of this API which only exposes the same information available through document.cookie. At this time we're not ready to expose ServiceWorkerRegistration.cookies or implement the CookieStoreManager as proposed; we will discuss this along with the subset proposal in the appropriate standards group",
```
Comment (zcorpan) [inline activities.json:475]: Please outline the privacy concerns inline or remove "Due to privacy concerns as outlined in issue 94,"

Comment (zcorpan) [inline activities.json:475]:
```suggestion
    "mozPositionDetail": "This API provides better access to cookies. Due to privacy concerns as outlined in issue 94, we are positive for a subset of this API which only exposes the same information available through document.cookie. At this time we're not ready to expose ServiceWorkerRegistration.cookies or implement the CookieStoreManager as proposed; we will discuss this along with the subset proposal in the appropriate standards group. See <a href=\"https://github.com/WICG/cookie-store/issues/241\">WICG/cookie-store issue #241</a>.",
```

**Review by zcorpan — CHANGES_REQUESTED** [review]

File: activities.json [hunk 2]
```diff
@@ -471,8 +471,8 @@
     "description": "An asynchronous Javascript cookies API for documents and workers",
     "id": "cookie-store",
     "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1475599",
-    "mozPosition": "defer",
-    "mozPositionDetail": "This API provides better access to cookies. However, the improvements also expand access to cookie metadata and the interactions with privacy features like the Storage Access API are unclear. Work on improving cookie interoperability, which is ongoing, might be needed before an assessment can be made.",
+    "mozPosition": "positive",
+    "mozPositionDetail": "This API provides better access to cookies. Due to privacy concerns as outlined in issue 94, we are positive for a subset of this API which only exposes the same information available through document.cookie. At this time we're not ready to expose ServiceWorkerRegistration.cookies or implement the CookieStoreManager as proposed; we will discuss this along with the subset proposal in the appropriate standards group. See <a href=\"https://github.com/WICG/cookie-store/issues/241\">WICG/cookie-store issue #241</a>.",
```
Comment (zcorpan) [inline activities.json:475]:
```suggestion
    "mozPositionDetail": "This API provides better access to cookies. We are positive for a subset of this API which only exposes the same information available through document.cookie. At this time we're not ready to expose ServiceWorkerRegistration.cookies or implement the CookieStoreManager as proposed; we will discuss this along with the subset proposal in the appropriate standards group. See <a href=\"https://github.com/WICG/cookie-store/issues/241\">WICG/cookie-store issue #241</a>.",
```

**Review by zcorpan — APPROVED** [review]

**Comment by zcorpan** [comment]
Rebased. Still needs r+ from @tantek

**Review by tantek — APPROVED** [review]
Updated rationale LGTM and thanks for linking the cookie-store issue to push this forward in the direction we prefer.

**Pull request merged**